AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -72,7 +73,7 @@ struct AirbnbSwiftFormatTool: ParsableCommand {
     }
 
     if !lint {
-      arguments += ["--autocorrect"]
+      arguments += ["--fix"]
```

Comment:
The docs say the options `--fix` and `--autocorrect` are the same. Did you notice a difference?
<img width="1602" alt="Screen Shot 2022-07-20 at 5 39 28 PM" src="https://user-images.githubusercontent.com/1791049/180105694-78939532-3540-40dd-9484-2a9e00006e07.png">

Reply:
I tried both and didn't notice a difference